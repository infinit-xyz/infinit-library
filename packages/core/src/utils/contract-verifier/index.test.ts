import { describe, expect, test } from 'vitest'

import * as index from './index'

describe('utils/verifier/index.ts', () => {
  test('export: should match snapshot', () => {
    expect(Object.keys(index)).toMatchSnapshot()
  })
})
