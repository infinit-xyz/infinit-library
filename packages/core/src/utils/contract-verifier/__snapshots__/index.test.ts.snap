// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`utils/verifier/index.ts > export: should match snapshot 1`] = `
[
  "BaseContractVerifier",
  "verifyContract",
  "BlockExplorerParams",
  "ContractInfo",
  "ContractVerifierCallback",
]
`;
